src/couch_rate/test/exunit/couch_rate_limiter_test.exs

```diff
@@ -0,0 +1,340 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
```

Comment:
I'm getting the following errors:

```
** (Mix) mix format failed due to --check-formatted.
The following files were not formatted:

  * src/couch_rate/test/exunit/couch_rate_limiter_test.exs
  * src/couch_rate/test/exunit/couch_rate_config_test.exs
```

```
  Refactoring opportunities                                                                                                                                                  
┃ 
┃ [F] → Pipe chain should start with a raw value.
┃       src/couch_rate/test/exunit/couch_rate_limiter_test.exs:319 #(Couch.Rate.Limiter.Test.to_str)
┃ [F] → Pipe chain should start with a raw value.
┃       src/couch_rate/test/exunit/couch_rate_limiter_test.exs:307 #(Couch.Rate.Limiter.Test.format_stats)
┃ [F] → Pipe chain should start with a raw value.
┃       src/couch_rate/test/exunit/couch_rate_limiter_test.exs:260 #(Couch.Rate.Limiter.Test.format_table)
┃ [F] → Pipe chain should start with a raw value.
┃       src/couch_rate/test/exunit/couch_rate_limiter_test.exs:253 #(Couch.Rate.Limiter.Test.format_table)
┃ [F] → Pipe chain should start with a raw value.
┃       src/couch_rate/test/exunit/couch_rate_limiter_test.exs:251 #(Couch.Rate.Limiter.Test.format_table)
┃  ...  (3 more, use `-a` to show them)
```

Could you add these tests to `make check-fdb`

Reply:
fixed in https://github.com/apache/couchdb/pull/2662/commits/cbad2ee2391eb60eca3593c61eb37f8c755a8190